### PR TITLE
Set Default statue of Delete Old Version Media option.

### PR DESCRIPTION
**Pull request by andyrogers2012**
Will resolve https://github.com/xibosignage/xibo/issues/201

### CODE REVIEW

File: install/database/86.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+INSERT INTO 'SETTING' ('settingid','setting','value','fieldType','helptext','options','cat','userChange','title','type','validation','ordering','default','userSee') VALUES
```
Comment (dasgarner) [inline install/database/86.sql]: You still have setting ID in there

File: install/master/data.sql [hunk 2]
```diff
@@ -272,6 +272,7 @@ INSERT INTO `setting` (`settingid`, `setting`, `value`, `fieldType`, `helptext`,
 (79, 'STS_TTL', '600', 'text', 'The Time to Live (maxage) of the STS header expressed in minutes.', NULL, 'network', 1, 'STS Time out', '', 90, '600', 1, 'int'),
 (80, 'MAINTENANCE_ALERTS_FOR_VIEW_USERS', '0', 'checkbox', 'Email maintenance alerts for users with view permissions to effected Displays.', NULL, 'displays', 1, 'Maintenance Alerts for Users', '', 60, '0', 1, 'checkbox'),
 (81, 'CALENDAR_TYPE', 'Gregorian', 'dropdown', 'Which Calendar Type should the CMS use?', 'Gregorian|Jalali', 'regional', 1, 'Calendar Type', '', 50, 'Gregorian', 1, 'string');
+(,'LIBRARY_MEDIA_DELETEOLDVER_CHECKB','Unchecked','dropdown','Default the checkbox for Deleting Old Version of media when a new file is being uploaded to the library.','Checked|Unchecked','defaults',1,'Default Delete Old Version of media','dropdown','',50,'Unchecked',1);
```
Comment (dasgarner) [inline install/master/data.sql]: You can have the ID in `data.sql` - this is on a fresh installation

File: locale/dbtranslate.php [hunk 3]
```diff
@@ -124,7 +124,7 @@
 echo __('Email maintenance alerts for users with view permissions to effected Displays.');
 echo __('Set the level of logging the CMS should record. In production systems "error" is recommended.');
 echo __('Which Calendar Type should the CMS use?');
-
+echo __('Default Delete Old Version of media');
```
Comment (dasgarner) [inline locale/dbtranslate.php]: Can this be:

Default for "Delete old version of Media" check box. Shown when Editing Library Media.

**Comment by dasgarner** [comment]
Hi Andy,

I've made some more comments - sorry to be picky!

Thanks,
Dan

File: install/database/86.sql (hunk 1 above)
Comment (andyrogers2012) [inline install/database/86.sql]: Sorry about that, I had removed the value but the comment name.  Still learning but am getting their.

File: install/master/data.sql (hunk 2 above)
Comment (andyrogers2012) [inline install/master/data.sql]: Added ID back into data.sql thanks

File: locale/dbtranslate.php (hunk 3 above)
Comment (andyrogers2012) [inline locale/dbtranslate.php]: Of course it can be.  Its your software, so have updated the files appropriately.

**Comment by dasgarner** [comment]
Grand - merged in!

Cheers,
DAn

**Pull request merged**